userapp/src/components/user/UserForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+import { makeStyles,
+    TextField,
+    Grid,
+}  from '@material-ui/core';
+
+  const useStyles = makeStyles((theme) => ({
+    root: {
+      '& .MuiFormControl-root': {
+        width: '80%',
+        margin: theme.spacing(1),
+      },
+      '& > *': {
+        margin: theme.spacing(1),
+      },
+    },
+  }));
+
+/**
+ * @return {JSX} UserCreate site
+ * routed at /user/new
+ */
+const UserForm = ({ initialUser }) => {
+    const [user, setUser] = useState(initialUser);
+    const classes = useStyles();
+    
+
+    const handleInputChange = ({ target: { name, value } }) => {
+        setUser({ ...user, [name]: value })
+    }
+
+    return (
+    <div>
+        <form className={classes.root} autoComplete="off" >
+         {/* onSubmit={handleSubmit}> */}
+            <Grid container>
+                <Grid item xs={3}>
+                    <TextField
```

Comment:
should these be from library "HelixTextField", etc

Reply:
Created! Library does not have it yet. Will tell Ian to store it in his npm library.